tests/org.eclipse.xtend.core.tests/src/org/eclipse/xtend/core/tests/validation/ValidationBug427432Test.xtend

```diff
@@ -41,7 +41,7 @@ class ValidationBug427432Test extends AbstractXtendTestCase {
 				}
 			
 			}
-		'''.parse.assertError(XbasePackage.Literals.XFEATURE_CALL, IssueCodes.DUPLICATE_CASE)
+		'''.parse.assertNoErrors
```

Comment:
My point was that we weren't able to detect all problematic cases already before. But ok, it makes sense to keep detecting at least those cases that can be realized with affordable effort.